pycalphad/core/minimizer.pyx

```diff
@@ -1009,6 +1014,8 @@ cdef bint remove_and_consolidate_phases(SystemSpecification spec, SystemState st
                 state.chemical_potentials[comp_idx] = spec.initial_chemical_potentials[comp_idx]
         else:
             state.free_stable_compset_indices = np.array(sorted(set(state.free_stable_compset_indices) - compset_indices_to_remove), dtype=np.int32)
+            for idx in compset_indices_to_remove:
+                state.times_compset_removed[idx] += 1
```

Comment:
My understanding here is that the only times `state.times_compset_removed` is incremented is
- here, when a composition set is removed due to being unstable or converging with another composition set in site fraction/composition space
- in `change_phases` to obey Gibbs phase rule
Is that correct? Just want to make sure I'm reading it right and that this counter isn't double counting.